cfgmgr/teammgr.cpp

```diff
@@ -108,6 +112,79 @@ void TeamMgr::doTask(Consumer &consumer)
     }
 }
 
+
+pid_t TeamMgr::getTeamPid(const string &alias)
+{
+    SWSS_LOG_ENTER();
+    pid_t pid = 0;
+
+    string file = string(PID_FILE_PATH) + alias + string(".pid");
+    ifstream infile(file);
+    if (!infile.is_open())
+    {
+        SWSS_LOG_WARN("The LAG PID file: %s is not readable", file.c_str());
+        return 0;
+    }
+
+    string line;
+    getline(infile, line);
+    if (line.empty())
+    {
+        SWSS_LOG_WARN("The LAG PID file: %s is empty", file.c_str());
+    }
+    else 
+    {
+        /*Store the PID value */
+        pid = stoi(line, nullptr, 10);
+    }
+
+    /* Close the file and return */
+    infile.close();
+
+    return pid;
+}
+
+
+void TeamMgr::addLagPid(const string &alias)
+{
+    SWSS_LOG_ENTER();
+    m_lagPIDList[alias] = getTeamPid(alias);
+}
+
+void TeamMgr::removeLagPid(const string &alias)
+{
+    SWSS_LOG_ENTER();
+    m_lagPIDList.erase(alias);
+}
+
+void TeamMgr::cleanTeamProcesses(int signo)
+{
+    pid_t pid = 0;
+
+    for (const auto& it: m_lagList)
+    {
+        pid = m_lagPIDList[it];
+        if(!pid) {
+            SWSS_LOG_WARN("Invalid PID found for LaG %s ", it.c_str());
+
+            /* Try to get the PID again */
+            pid = getTeamPid(it);
+        }
+
+        if(pid > 0)
+        {
+            SWSS_LOG_WARN("Sending Signal to (PID: %d) for LaG %s ", pid, it.c_str());
+            kill(pid, signo);
+        }
+        else
+        {
+            SWSS_LOG_WARN("Cound not find PID corresponding to LaG %s ", it.c_str());
```

Comment:
Can you please make it as an error?
Also It's better to say "Can't send TERM signal to LAG %s. PID wasn't found"